task_1/task_1/EncodeDecode.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace program
+{
+    public static class EncodeDecode
+    {
+        //encodes bytes
```

Comment:
```suggestion
        /// <summary>
        /// Method for encoding input byte array.
        /// </summary>
        /// <param name="bytes">Byte array for incoding.</param>
        /// <returns>Incoded byte array and incoding ratio.</returns>
```
Вот так нужно оформлять комментарии для методов.